src/pages/layer-2.js

```diff
@@ -284,92 +283,113 @@ const Layer2Page = ({ data }) => {
+      title: translateMessageId("layer-2-lower-fees-title", intl),
+      description: translateMessageId("layer-2-lower-fees-description", intl),
     },
     {
       emoji: ":closed_lock_with_key:",
-      className: "security-card",
-      title: "Maintain security",
-      description:
-        "Layer 2 blockchains settle their transactions on Ethereum Mainnet, allowing users to benefit from the security of the Ethereum network.",
+      title: translateMessageId("layer-2-maintain-security-title", intl),
+      description: translateMessageId(
+        "layer-2-maintain-security-description",
+        intl
+      ),
     },
     {
       emoji: ":hammer_and_wrench:",
-      title: "Expand use cases",
-      description:
-        "With higher transactions per second, lower fees, and new technology, projects will expand into new applications with improved user experience.",
+      title: translateMessageId("layer-2-expand-use-cases-title", intl),
+      description: translateMessageId(
+        "layer-2-expand-use-cases-description",
+        intl
+      ),
     },
   ]
 
   const rollupCards = [
     {
       image: getImage(data.optimisticRollup),
-      title: "Optimistic rollups",
-      description:
-        "Optimistic rollups are 'optimistic' in the sense that transactions are assumed to be valid, but can be challenged if necessary. If an invalid transaction is suspected, a fault proof is ran to see if this has taken place.",
-      childSentence: "More on optimistic rollups",
+      title: translateMessageId("layer-2-optimistic-rollups-title", intl),
+      description: translateMessageId(
+        "layer-2-optimistic-rollups-description",
+        intl
+      ),
+      childSentence: translateMessageId(
+        "layer-2-optimistic-rollups-childSentance",
+        intl
+      ),
       childLink: "/developers/docs/scaling/optimistic-rollups/",
     },
     {
       image: getImage(data.zkRollup),
-      title: "Zero-knowledge rollups",
-      description:
-        "Zero-knowledge rollups use validity proofs where transactions are computed off-chain, and then compressed data is supplied to Ethereum Mainnet as a proof of their validity.",
-      childSentence: "More on zk-rollups",
+      title: translateMessageId("layer-2-zk-rollups-title", intl),
+      description: translateMessageId("layer-2-zk-rollups-description", intl),
+      childSentence: translateMessageId(
+        "layer-2-zk-rollups-childSentance",
+        intl
+      ),
       childLink: "/developers/docs/scaling/zk-rollups/",
     },
   ]
 
   const toolsData = {
     information: [
       {
-        title: "L2BEAT",
-        description:
-          "L2BEAT is a great resource for looking at technical risk assessments of layer 2 projects. We recommend checking out their resources when researching specific layer 2 projects.",
+        title: translateMessageId("layer-2-tools-l2beat-title", intl),
```

Comment:
Typically we do not set product names up for translation. Some of these names, like "L2 Fees" I suppose are debatable, but many of these I would assume would remain untranslated, such as L2BEAT, Zepper, Zerion, etc.
<img width="331" alt="image" src="https://user-images.githubusercontent.com/54227730/170547401-1de1709e-452c-441e-8b9e-10135396b9bf.png">
Would suggest we revert these six strings.